assets/js/phoenix_live_view/hooks.js

```diff
@@ -58,8 +58,8 @@ let Hooks = {
 }
 
 let findScrollContainer = (el) => {
+  if (["HTML", "body"].indexOf(el.nodeName) >= 0) return null
```

Comment:
I'm quite sure nodeName for body is uppercase as well. Also a comment might be nice:

```suggestion
  // the scroll event won't be fired on the html/body element even if overflow is set
  // therefore we return null to instead listen for scroll events on document
  if (["HTML", "BODY"].includes(el.nodeName)) return null
```